application-services/custom/camera-management/README.md

```diff
@@ -1,19 +1,117 @@
+   latest: Pulling from library/hello-world
+   2db29710123e: Pull complete 
+   Digest: sha256:10d7d58d5ebd2a652f4d93fdd86da8f265f5318c6a73cc5b6a9798ff6d2b2e67
+   Status: Downloaded newer image for hello-world:latest
+
+   Hello from Docker!
+   This message shows that your installation appears to be working correctly.
+   ...
+   ```
+
+### Install Docker Compose
+Install Docker Compose from the official repository as documented on the [Docker Compose](https://docs.docker.com/compose/install/linux/#install-using-the-repository) site.
+
+### Install Tools
+Install the build, media streaming, and parsing tools:
+
+```bash
+sudo apt install build-essential
+```
+
 ## Steps for running this example:
-We expect you'll be running in a relatively modern Linux environment
-with `docker`, `docker-compose`, and `make` installed.
 
-1. Get the [EdgeX Core Services][edgex-core-services] and Edgex device ONVIF service running by referring to docs from
-   [Edgex Onvif device service][device-onvif-camera].
-2. Get [Edge Video Analytics Microservice][evam] running for inference.
+### 1. Start the EdgeX Core Services and Device Services.
+
+1. Clone `edgex-compose` from github.com.
+   ```shell 
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```  
+
+2. Navigate to the EdgeX `compose-compose` directory:
+
+   ```bash
+   cd edgex-compose
+   ```
+
+3. Checkout latest branch.
+   ```shell
+   git checkout levski
+   ```
+
+Note: The `levski` branch is the latest stable branch at the time of this update. 
+
+4. Navigate to the EdgeX `edgex-compose/compose-builder` directory:
+
+   ```bash
+   cd compose-builder/
+   ```
+
+5. Update the `add-device-usb-camera.yml` file:
+
+   a. Add the rtsp server hostname environmental variable to the `device-usb-camera` service.
+   ```yml
+   services:
+      device-usb-camera:
+             environment:
+               DRIVER_RTSPSERVERHOSTNAME: `your-local-ip-address`
+   ```
+      where `your-local-ip-address` is the ip address of the machine running the rtsp server.
+
+   b. Under the `ports` section, find the one for port 8554 and change the host_ip from 127.0.0.1 to either 0.0.0.0 or the ip address you put in the previous step.
+6. (Optional) If onvif cameras are being used, please refer to the [Edgex Onvif device service](https://github.com/edgexfoundry/device-onvif-camera/blob/main/doc/guides/CustomStartupGuide.md) documentation to run the onvif device service with camera credentials.
+
+7. Run the following `make` command to run the edgex core services along with the Onvif and Usb device services.
+```shell
+   make run no-secty ds-onvif-camera ds-usb-camera 
+```   
+
+### 2. Start [Edge Video Analytics Microservice](https://www.intel.com/content/www/us/en/developer/articles/technical/video-analytics-service.html) running for inference.
+
+Note: the port for EVAM result streams has been changed from 8554 to 8555 to avoid conflicts with device-usb-camera service.
```

Comment:
```suggestion
> **Note:** the port for EVAM result streams has been changed from 8554 to 8555 to avoid conflicts with device-usb-camera service.
```